test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -277,9 +277,9 @@ public HubConnectionTests(ServerFixture<Startup> serverFixture, ITestOutputHelpe
                     var cts = new CancellationTokenSource();
                     cts.Cancel();
 
-                    var channel = await connection.StreamAsync<int>("Stream", 5, cts.Token).OrTimeout();
+                    var canceledStreamTask = connection.StreamAsync<int>("Stream", 5, cts.Token).OrTimeout();
```

Comment:
Exceptions and cancellations should go through the channel. The previous behavior was correct.